examples/guides/services/healthcare-service/.gitignore

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
Shall we  merge this with main .gitginore?

Reply:
Done